solutions/web3-fetching-strategies/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "vercel-example--web3-fetching-strategies",
+  "version": "1.0.0",
+  "repository": "https://github.com/vercel/examples.git",
+  "license": "MIT",
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "contract-factories": "typechain --target=ethers-v5  ./lib/BAYC.abi.json"
```

Comment:
What's this? if it's needed for something it should be documented in the readme

Reply:
This was to showcase some convenience package and was a bit out of scope.

I removed it but I put a link to it in the example!